Sources/Benchmark/BenchmarkSetting.swift

```diff
@@ -14,29 +14,58 @@
 
 public enum BenchmarkSetting {
     case iterations(Int)
+    case warmupIterations(Int)
+    case filter(String)
+    case allowDebugBuild(Bool)
```

Comment:
I'm not sure that this is reasonable as a benchmark setting, as if it's set on a given benchmark, I don't think it'll let you actually run it in a debug build, right?

Reply:
That's a great point, I removed this setting 1d7b0c8.